From a71c8f9e5d76f5c257ed6f24db3eda31563b10a2 Mon Sep 17 00:00:00 2001
From: Matheus Paschoal <paschoal@gmail.com>
Date: Tue, 26 Aug 2025 20:58:05 -0400
Subject: patch-2

---
 src/py/libcamera/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/py/libcamera/meson.build b/src/py/libcamera/meson.build
index 33ab6579..52531aa7 100644
--- a/src/py/libcamera/meson.build
+++ b/src/py/libcamera/meson.build
@@ -18,7 +18,7 @@ gen_py_controls = files('gen-py-controls.py')
 pycamera_sources += custom_target('py_gen_controls',
                                   input : controls_files,
                                   output : ['py_controls_generated.cpp'],
-                                  command : [gen_py_controls, '--mode', 'controls', '-o', '@OUTPUT@',
+                                  command : ['python3', gen_py_controls, '--mode', 'controls', '-o', '@OUTPUT@',
                                              '-t', gen_py_controls_template, '@INPUT@'],
                                   depend_files : [py_mod_controls],
                                   env : py_build_env)
@@ -26,7 +26,7 @@ pycamera_sources += custom_target('py_gen_controls',
 pycamera_sources += custom_target('py_gen_properties',
                                   input : properties_files,
                                   output : ['py_properties_generated.cpp'],
-                                  command : [gen_py_controls, '--mode', 'properties', '-o', '@OUTPUT@',
+                                  command : ['python3', gen_py_controls, '--mode', 'properties', '-o', '@OUTPUT@',
                                              '-t', gen_py_controls_template, '@INPUT@'],
                                   depend_files : [py_mod_controls],
                                   env : py_build_env)
-- 
2.50.1

