--- ../st/config.def.h	2025-05-09 11:47:29.954048455 -0400
+++ ./config.def.h	2025-05-09 12:05:43.990904508 -0400
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "Iosevka:pixelsize=16:antialias=true:autohint=true";
+static int borderpx = 8;
 
 /*
  * What program is execed by st depends of these precedence rules:
